packages/command-palette/src/tracks.ts

```diff
@@ -0,0 +1,8 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+
+export const recordCommandPaletteOpen = ( currentRoute: string, openType: string ) => {
```

Comment:
```suggestion
export const recordCommandPaletteOpen = ( currentRoute: string, openType: 'keyboard' | 'sidebar_click' ) => {
```

If the `openType` list is not huge, I'd rather have it typed to prevent typos. Wdyt?